cluster-scope/base/operators.coreos.com/subscriptions/openshift-pipelines-operator-rh/subscription.yaml

```diff
@@ -4,7 +4,7 @@ kind: Subscription
 metadata:
   name: openshift-pipelines-operator-rh
 spec:
-  channel: DEFINED_IN_OVERLAY
```

Comment:
Any reason we are not just setting this in the OSC overlays? Do no other overlays use different channels?

Reply:
I just set it in the base since it's the same for both clusters. I don't see a reason to patch it separately. Less work imo.